src/desktop/DesktopMain.ts

```diff
@@ -90,7 +92,25 @@ type Components = {
 }
 const tfs = new TempFs(fs, electron, cryptoFns)
 const desktopUtils = new DesktopUtils(process.argv, tfs, electron)
-const desktopCrypto = new DesktopNativeCryptoFacade(fs, cryptoFns, tfs)
+const secretStorage = new KeytarSecretStorage()
+const argon2 = new WASMArgon2idFacade(async () => {
+	// this is kinda prone to deadlock if we use argon2 before
+	// conf is ready since wasm -> conf -> crypto -> wasm in an async way
+	const webAssetsPath = await conf.getConst(BuildConfigKey.webAssetsPath)
```

Comment:
`getConst` should never need encryption. It is arguably completely independent from persistence of `DesktopConfig`, except that it gets defaults from the build config (should it just be hardcoded in the code? maybe)

Reply:
It's more of a theoretical possibility. Storing the build constants in the ts-code might make the build more difficult though - maybe we need to think about splitting the config handling at some point.

Reply:
I am not talking about all the constants, just about the default values for the config, my opinion is that they are not variable and have no business being in the build code.